src/System.ComponentModel.Annotations/tests/System/ComponentModel/DataAnnotations/MetadataTypeAttributeTests.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.ComponentModel.DataAnnotations.Tests
+{
+    public class MetadataTypeAttributeTests
+    {
+        
```

Comment:
Missing tests?

Reply:
I was unable to find any test for this in either .net framework & mono repositories.
All tests scenarios are welcomed, I also try to design some scenarios for that too.

Reply:
Yes you'll have to write some but hopefully should be straightforward